app/domain/authentication/authn_jwt/signing_key/fetch_signing_key_parameters_from_variables.rb

```diff
@@ -0,0 +1,52 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # This class is responsible for fetching values of all variables related
+      # to signing key settings area
+      FetchSigningKeyParametersFromVariables ||= CommandClass.new(
+        dependencies: {
+          check_authenticator_secret_exists: Authentication::Util::CheckAuthenticatorSecretExists.new,
+          fetch_authenticator_secrets: Authentication::Util::FetchAuthenticatorSecrets.new
+        },
+        inputs: %i[authenticator_input]
```

Comment:
Maybe consider using delgators for
accout, service_id, authenticator_name

Reply:
From one point of view let it be.
From other do we have some rule of thumb when to use delegators?